caas/kubernetes/provider/k8s.go

```diff
@@ -817,6 +817,31 @@ func (k *kubernetesClient) DeleteOperator(appName string) (err error) {
 	return errors.Trace(k.deleteDeployment(operatorName))
 }
 
+func getLoadBalancerAddress(svc *core.Service) string {
+	// reason we have this is because different cloud provider
+	// has different way to report back LB address.
+	// This should cover most of case now.
```

Comment:
different cloud providers have a different way to report back the Load Balancer address.
This covers the cases we know about so far.